README.md

```diff
@@ -43,19 +43,38 @@ The basic sequence for driving the Bulk API is:
 
 ## Bulk Query
 
-`SalesforceBulk.create_query_job(object_name, contentType='CSV', concurrency=None)`
+`bulk.create_query_job(object_name, contentType='CSV')`
 
 Example
 
 ```
+import unicodecsv
 job = bulk.create_query_job("Contact", contentType='CSV')
 batch = bulk.query(job, "select Id,LastName from Contact")
-while not bulk.is_batch_done(job, batch):
+bulk.close_job(job)
+while not bulk.is_batch_done(batch):
 	sleep(10)
+
+for result in bulk.get_all_results_for_query_batch(batch):
+	with unicodecsv.DictReader(result, encoding='utf-8') as reader:
```

Comment:
I got an error when I ran this using Python 2.7. I had to change it to `reader = unicodecsv.DictReader(result, encoding='utf-8')`.